### DISCUSSION TITLE
Local cache?

### DISCUSSION THREAD

**Discussion by wayneyaoo** [jeffvli/feishin#123] [Q&A]
I couldn't find the feature in the roadmap, but does it have any plan to implement local cache of the audio file? I hosted my Jellyfin behind a proxy that looks at how much data is passed through. Appreciate it if it's considered to have this local cache feature in the future! thanks! : )

**Comment by CMAYK** [comment]
I second this!
Symphonium on mobile maintains a local cache and it is super helpful, and especially nice for large libraries and loading times

I noticed Feishin will spend a lot of time buffering, especially on a slow internet connection because it is trying to download the necessary data. Yet it doesn't seem to cache anything for quickly loading locally.
